src/routes/transactions/mappers/queued-items/queued-items.mapper.ts

```diff
@@ -132,4 +135,23 @@ export class QueuedItemsMapper {
         },
     );
   }
+
+  /**
+   * Adjusts the timestamps of transactions array by given offset
+   * @param timestamp transactions to offset the timestamp of
+   * @param timezoneOffset UTC timezone offset in milliseconds
+   */
+  private getTimezoneOffsetTransactions(
+    transactions: MultisigTransaction[],
+    timezoneOffset: number,
+  ): MultisigTransaction[] {
+    if (timezoneOffset === 0) {
+      return transactions;
+    }
+
+    return transactions.map((transaction) => {
+      transaction.submissionDate.setUTCMilliseconds(timezoneOffset);
+      return transaction;
+    });
```

Comment:
Good catch, I've updated this accordingly in 1d584755bf261c993d2894866c13156f671230de.

> This might be an hint to have a subset of MultisigTransaction with properties that we know are only part of a queued transaction

This is something to consider. I noticed that the `multisigTransactionBuilder`, for instance, always sets the `executionDate`. A subset would help avoid this.